doc/lightning-commando-rune.7.md

```diff
@@ -193,6 +193,9 @@ On success, an object is returned, containing:
 - **rune** (string): the resulting rune
 - **unique_id** (string): the id of this rune: this is set at creation and cannot be changed (even as restrictions are added)
 
+The following warnings may also be returned:
+- **warning_usage**: A warning shown when runes are created with powers that could drain your node
```

Comment:
`warning_unrestricted_rune`?  Warning names need to be as explicit as possible...

Reply:
:+1: done